xc/device.ml

```diff
@@ -671,7 +675,6 @@ module Vif = struct
     let backend = { domid = backend_domid; kind = Vif; devid = devid } in
     let device = { backend = backend; frontend = frontend } in
 
-    let mac = Mac.check_mac mac in
```

Comment:
Huh?

Reply:
This is moved to `xc/xenops_server_xen.ml` as this is common code for both Vif and Sriov.

Reply:
This is where it would have been a lot better to have smaller commits: a single commit to refactor this part including moving this check.

Reply:
Sure in next time:)